systemd/osdf-director.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description = Pelican service osdf-director
+After = network.target nss-lookup.target
+
+[Service]
+EnvironmentFile = -/etc/sysconfig/osdf-director
+ExecStart = /usr/bin/osdf --config /etc/pelican/osdf-director.yaml director serve
+Restart = on-failure
+RestartSec = 20s
```

Comment:
I see your commit removing the working dir for the director/registry but it's not clear to me why they don't need it

Reply:
I suppose the director/registry can core dump too.

Most service files don't have a WorkingDirectory set; either the services run as root (in which case they can write their core files to `/`), or they run as non-root (in which case they start in their home directories and can write core files there). The reason we have to specify it in this case is that pelican starts as root but then spawns xrootd with reduced privileges (without changing the directory first).